spring-rabbit/src/test/java/org/springframework/amqp/rabbit/listener/MessageListenerContainerLifecycleIntegrationTests.java

```diff
@@ -115,7 +119,7 @@ public int value() {
 		}
 	}
 
-	@Rule
+//	@Rule
```

Comment:
We're now not acking processed messages during shutdown - we need to take this action only after we have given up and forced the close.